src/main/webapp/js/components/auth/login.jsx

```diff
@@ -23,8 +24,11 @@ const validate = (values) => {
 
 class Login extends Component {
   onSubmit = (values) => {
-    const { history } = this.props;
-    this.props.signinUser(values, () => {
+    const { history, savedLogins } = this.props;
+    this.props.signinUser({
+      ...values,
+      savedLogin: _.get(savedLogins, values.username, null),
+    }, () => {
       history.push('/');
```

Comment:
I think instead of this callback you can render the redirect if user is authenticated